### PR TITLE
Enable connection draining for the API ELBs

### PR DESCRIPTION
**Pull request by alexsomesan**
@ethernetdan noticed strange behaviour on the Kubelets when master nodes are being scaled.

This might be due to the fact that nodes are being removed from the API ELB while having ongoing connections from kubelets.

### CODE REVIEW

**Comment by s-urbaniak** [comment]
@alexsomesan Destroy failed with the following failure:

```

* module.vpc.aws_internet_gateway.igw (destroy): 1 error(s) occurred:

* aws_internet_gateway.igw: Error waiting for internet gateway (igw-afe433c8) to detach: couldn't find resource (31 retries)
```

**Comment by Quentin-M** [comment]
ok to test

**Comment by Quentin-M** [comment]
I added retries on destroy. Let's see!

**Comment by Quentin-M** [comment]
AWS timed-out? Let me try again, othewise I will increase that value. Yesterday they were all taking <20min. Maybe the region is getting more busy during the work day?

**Comment by Quentin-M** [comment]
ok to test

**Comment by alexsomesan** [comment]
retest this please

**Comment by Quentin-M** [comment]
ok to test

**Comment by s-urbaniak** [comment]
retest this please

**Comment by s-urbaniak** [comment]
@alexsomesan do you mind to rebase on master?

**Comment by alexsomesan** [comment]
retest this please
